BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -1321,14 +1321,30 @@ IEnumerator setVar(string name, float value)
             yield return new WaitForFixedUpdate();
             typeof(BDModulePilotAI).GetField(name).SetValue(this, value);
         }
-
+        float timer = 0;
         void FixedUpdate()
         {
             //floating origin and velocity offloading corrections
             if (!FloatingOrigin.Offset.IsZero() || !Krakensbane.GetFrameVelocity().IsZero())
             {
                 if (lastTargetPosition != null) lastTargetPosition -= FloatingOrigin.OffsetNonKrakensbane;
             }
+            if (weaponManager.detectedTargetTimeout > weaponManager.targetScanInterval)
+            {
+                timer += Time.fixedDeltaTime;
+                if (timer >= 50) //add some error to the predicted position every second
+                {
+                    Vector3 staleTarget = new Vector3();
+                    staleTarget.x = UnityEngine.Random.Range(0f, (float)lastKnownVector.magnitude / 10);
+                    staleTarget.y = UnityEngine.Random.Range(0f, (float)lastKnownVector.magnitude / 10);
+                    staleTarget.z = UnityEngine.Random.Range(0f, (float)lastKnownVector.magnitude / 20);
+                    timer = 0;
```

Comment:
Could perhaps play around with the values for more/less target position decay; +/- ~20-30m/s isn't all that much, even over 30 seconds